.github/e2e-tests.yml

```diff
@@ -257,7 +257,6 @@ runner-test-matrix:
     test_env_type: docker
     runs_on: ubuntu-latest
     triggers:
-      - PR E2E Core Tests
       - Merge Queue E2E Core Tests
```

Comment:
Should we also remove `Merge Queue E2E Core Tests` for this test? Otherwise it can fail in MQ when it could've failed (consistently) in the PR.